### PR TITLE
New derived table inst alt titles

### PR DESCRIPTION
**Pull request by jenmawe**
New metadb derived table for instance derived tables. I also added this to the runlist.txt file.

### CODE REVIEW

**Comment by jenmawe** [comment]
closes #525

File: sql_metadb/derived_tables/instance_alternative_titles.sql [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+DROP TABLE IF EXISTS instance_alternative_titles;
+
+CREATE TABLE instance_alternative_titles AS 
+SELECT 
+	i.id AS instance_id,
+	jsonb_extract_path_text(i.jsonb, 'hrid') AS instance_hrid,
+	jsonb_extract_path_text(titles.jsonb, 'alternativeTitle') AS alternative_title,
+    jsonb_extract_path_text(titles.jsonb, 'alternativeTitleTypeId')::uuid AS alternative_title_type_id,
+    titles.ordinality AS alternative_title_ordinality,
+    jsonb_extract_path_text(att.jsonb, 'name') AS alternative_type_name
```
Comment (lib-eric) [inline sql_metadb/derived_tables/instance_alternative_titles.sql:13]:
```suggestion
    jsonb_extract_path_text(att.jsonb, 'name') AS alternative_title_type_name
```
To match with name in INDEX and to be consistent with naming.

**Review by lib-eric — CHANGES_REQUESTED** [review]
- [x] Query runs
- [x] Data output makes sense
- [x] Data types look correct
- [x] Query logic seems correct
- [ ] Column names make sense and are spelled correctly
	See line comment/suggested change (line 13 update select field name)
- [x] Query syntax and style are appropriate (e.g., correct use of double quotes, lower case for data types)

For derived tables:
- [x] query has indexes on columns
- [x] vacuum analyze has been added to the bottom
- [x] table appears in runlist.txt at an appropriate point

File: sql_metadb/derived_tables/instance_alternative_titles.sql (hunk 1 above)
Comment (jenmawe) [reply sql_metadb/derived_tables/instance_alternative_titles.sql]: Thanks Eric. I just made that change.

**Review by lib-eric — APPROVED** [review]

**Pull request merged**